imx6ul-var-dart machine specific integration of resin environment configuration.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Florin Sarbu <florin@resin.io>

Index: git/include/configs/mx6ul_var_dart.h
===================================================================
--- git.orig/include/configs/mx6ul_var_dart.h
+++ git/include/configs/mx6ul_var_dart.h
@@ -28,6 +28,10 @@
 
 #undef CONFIG_LDO_BYPASS_CHECK
 
+/* We need the part command */
+#define CONFIG_PARTITION_UUIDS
+#define CONFIG_CMD_PART
+
 /* SPL options */
 #define CONFIG_SPL_LIBCOMMON_SUPPORT
 #define CONFIG_SPL_MMC_SUPPORT
@@ -115,7 +119,7 @@
 	"mmcroot=" CONFIG_MMCROOT " rootwait rw\0" \
 	"mmcautodetect=yes\0" \
 	"mmcargs=setenv bootargs console=${console},${baudrate} " \
-		"root=${mmcroot} ${cma_size}\0" \
+		"${resin_kernel_root} rootwait rw ${cma_size}\0" \
 	"loadbootscript=" \
 		"fatload mmc ${mmcdev}:${mmcpart} ${loadaddr} ${script};\0" \
 	"bootscript=echo Running bootscript from mmc ...; " \
@@ -150,6 +154,10 @@
 #else
 #define BOOT_ENV_SETTINGS	MMC_BOOT_ENV_SETTINGS
 #define CONFIG_BOOTCOMMAND \
+	"setenv resin_kernel_load_addr ${loadaddr};" \
+	"run resin_set_kernel_root;" \
+	"setenv mmcdev ${resin_mmcdev};" \
+	"setenv mmcpart ${resin_boot_part};" \
 	"mmc dev ${mmcdev};" \
 	"mmc dev ${mmcdev}; if mmc rescan; then " \
 		"if run loadbootscript; then " \
